libraries/core-react/src/Typography/Typography.tsx

```diff
@@ -81,7 +81,8 @@ const StyledTypography = styled.p<StyledProps>`
       }
     `}
 `
-type Props = {
+
+export type Props = {
```

Comment:
Exported to be able to read in `BannerMessage.tsx`, but not exported via `Typography/index.ts`.